pimpmycause/pimpmycause/templates/layout.html

```diff
@@ -43,9 +43,9 @@
                         </form>
                     {% else %}
                         <div class="navigation__login-social">
-                            <a href="#" class="navigation__social-icon">{% include "icons/linkedin.svg" %}</a>
-                            <a href="#" class="navigation__social-icon">{% include "icons/facebook.svg" %}</a>
-                            <a href="#" class="navigation__social-icon">{% include "icons/twitter.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/linkedin.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/facebook.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/twitter.svg" %}</a>
```

Comment:
This SVG seems to have only one path each, have you consider to use iconfont? They offer better compatibility and they have lower overhead on request, since you load the entire set in one http call instead of having one for each image.